.gitignore

```diff
@@ -1 +1,2 @@
+mytest/*
```

Comment:
I have made some time in my agenda and developed the BGP module quite a
lot. I'd appreciate some feedback.

Thanks, PA
El 21/1/2016 19:15, "Pedro Andres Aranda Gutierrez" <paaguti@gmail.com>
escribió:

> And what about the rest?
> El 21/1/2016 18:15, "Pierre Lalet" <notifications@github.com> escribió:
>
>> In .gitignore
>> <https://github.com/secdev/scapy/pull/10#discussion_r50431294>:
>>
>> > @@ -1 +1,2 @@
>> > +mytest/*
>>
>> Can you remove that?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/secdev/scapy/pull/10/files#r50431294>.
>>
>